core-job/src/main/java/org/apache/kylin/job/dao/ExecutableDao.java

```diff
@@ -391,4 +405,13 @@ public void deleteJobOutput(String uuid) throws PersistentException {
             throw new PersistentException(e);
         }
     }
+
+    public void reloadAll() throws IOException {
+        try (AutoReadWriteLock.AutoLock lock = executableDigestMapLock.lockForWrite()) {
+            executableDigestCrud.reloadAll();
```

Comment:
Why this is in "lockForWrite()" but the next reload is in "lockForRead"?

Reply:
@shaofengshi You are right, it's a mistake, should use lockForWrite during reloading. PR updated, thanks